airbyte-api/src/main/openapi/config.yaml

```
@@ -2982,6 +2982,13 @@ components:
           format: uri
         icon:
           type: string
+        normalizationRepository:
+          type: string
+        normalizationTag:
+          type: string
+        supportDbt:
+          type: boolean
+          default: false
```

```suggestion
        supportsDbt:
          type: boolean
          default: false
```

nit to keep consistent